libs/employees/src/lib/employees/employees.component.ts

```diff
@@ -1,38 +1,24 @@
 import { Component, OnInit } from '@angular/core';
+//import { Subscription } from 'rxjs';
 import { TableColumn } from '@razroo-fully-architected-dashboard/ui/common';
 
-const ELEMENT_DATA = [
-  {name: 'test', position: 'test position', location: 'New York',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'Los Angelos',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'Alabama',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-  {name: 'test', position: 'test position', location: 'new york',
-    permission: 'test permission', reportTo: 'no one'},
-];
-
 @Component({
   selector: 'razroo-fully-architected-dashboard-employees',
   templateUrl: './employees.component.html',
-  styleUrls: ['./employees.component.scss']
+  styleUrls: ['./employees.component.scss'],
+ 
 })
+
 export class EmployeesComponent implements OnInit {
-  elementData;
-  employeeTableColumns: TableColumn[];
+  
```

Comment:
In order to call the employeesService. Inside of your EmployeesComponent, we will go ahead and create a constructor i.e. 
```
constructor(private employeesService: EmployeesService){} 
```

Will need to import the EmployeesService of course, similar to how this file is importing `{Component, OnInit}`